src/nvme/types.h

```diff
@@ -4856,6 +4856,34 @@ enum nvmf_tcp_sectype {
 	NVMF_TCP_SECTYPE_TLS13	= 2,
 };
 
+/**
+ * enum nvmf_get_log_discovery_lid_support - Discovery log specific support
+ * @NVMF_LOG_DISC_LID_NONE:	None
+ * @NVMF_LOG_DISC_LID_EXTDLPES:	Extended Discovery Log Page Entries Supported
+ * @NVMF_LOG_DISC_LID_PLEOS:	Port Local Entries Only Supported
+ * @NVMF_LOG_DISC_LID_ALLSUBES:	All NVM Subsystem Entries Supported
+ */
+enum nvmf_get_log_discovery_lid_support {
+	NVMF_LOG_DISC_LID_NONE		= 0,
+	NVMF_LOG_DISC_LID_EXTDLPES	= (1 << 0),
+	NVMF_LOG_DISC_LID_PLEOS		= (1 << 1),
+	NVMF_LOG_DISC_LID_ALLSUBES	= (1 << 2),
+};
+
+/**
+ * enum nvmf_get_log_discovery_lsp - Discovery log specific field
+ * @NVMF_LOG_DISC_LSP_NONE:	None
+ * @NVMF_LOG_DISC_LSP_EXTDLPE:	Extended Discovery Log Page Entries
+ * @NVMF_LOG_DISC_LSP_PLEO:	Port Local Entries Only
+ * @NVMF_LOG_DISC_LSP_ALLSUBE:	All NVM Subsystem Entries
+ */
+enum nvmf_get_log_discovery_lsp {
```

Comment:
Same here, the '_get_' seems to be wrong here.

Reply:
Agreed. I will drop the `get`. 
FWIW, I was trying to stay consistent with another "get log" LSP enum (i.e. `nvme_cmd_get_log_telemetry_host_lsp`)

Reply:
Ah, good to know. I think we should rename that one for 2.x!